network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_index_callout_box.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags static i18n %}
+
+<div class="tw-h-full tw-flex tw-flex-col tw-justify-between tw-p-[36px] tw-border tw-border-black tw-shadow-[10px_10px_rgba(0,0,0)]">
+  <div id="callout-box-body" class="tw-mb-[4rem]">
+        <div class="tw-flex">
+          {% if value.related_topics %}
+            <div class="tw-mb-3">         
+              {% include '../fragments/blog_topics.html' with topics=value.related_topics %}
+            </div>
+          {% endif %}
+          {% if value.show_icon %}
+            <img src="{% static "_images/headphones-icon.svg" %}" alt="{% trans "audio" context "icon description for screen readers" %}" class="tw-h-4 tw-w-4 tw-mr-2 tw-my-1 tw-mb-3">
+          {% endif %}
+        </div>
+        <h1 class="tw-text-[40px] tw-mb-2">{{ value.title }}</h1>
```

Comment:
**Suggestion** We currently working on establishing a design system. To make that easier to maintain, we should avoid arbitrary values as much as possible. Try to use `tw-h2-heading` or one of the other heading styles to see what suits the design.

Reply:
Hi Tibor! Thanks for the suggestion for using tw-h2-heading, it turns out it was exactly what I needed! The only catch being that on smaller screens, the tw-h2-heading class reduces the fontsize, where the figma designs ask for the text size to be 40px on all screensizes. 

Due to this, I left the tw-text-[40px]